commands/utils/simplifiedoutput.go

```diff
@@ -13,14 +13,15 @@ const (
 )
 
 type SimplifiedOutput struct {
+	showCaColumn   bool
 	entitledForJas bool
```

Comment:
why we need entitledForJas?

Reply:
For the UntitledForJasMsg.
If the `showCaColumn` is false, that doesn't mean that `entitled` is false as well.